rails/client/app/bundles/Kaiju/components/Project/components/Layers/Layers.jsx

```diff
@@ -20,21 +20,7 @@ const propTypes = {
 };
 
 class Layers extends React.Component {
-  constructor() {
-    super();
-    this.generateTree = this.generateTree.bind(this);
-    this.scrollToComponent = this.scrollToComponent.bind(this);
-  }
-
-  componentDidMount() {
-    window.addEventListener('message', this.scrollToComponent);
-  }
-
-  componentWillUnmount() {
-    window.removeEventListener('message', this.scrollToComponent);
-  }
-
-  scrollToComponent({ data }) {
+  static scrollToComponent({ data }) {
```

Comment:
`Expected 'this' to be used by class method 'scrollToComponent'  class-methods-use-this`